packages/vue/src/components/molecules/SfRadio/SfRadio.vue

```diff
@@ -91,8 +91,8 @@ export default {
     },
   },
   methods: {
-    inputHandler() {
-      this.$emit("input", this.value);
+    changeHandler() {
+      this.$emit("change", this.value);
```

Comment:
If you change it like this, it will cause migration problem. Why not keep `input` emitter event and only do assign it to the `@change` native event?

Reply:
Please see my changes.

Reply:
@AdamPawlinski I meant we should keep both `inputHandler` and `changeHandler` since both are tied to two different types of events, then in template we should do
`@input="inputHandler" @change="changeHandler"`